signup/signup.js

```diff
@@ -0,0 +1,74 @@
+import { EMAIL_REGEX, TEST_EMAIL } from "/js/account/constant.js";
+import { eyeOpen, eyeClose } from "/js/account/passwordIconToggle.js";
+import { email, emailError } from "/js/account/email.js";
+import { password, passwordError } from "/js/account/password.js";
+
+const passwordConfirm = document.querySelector("#passwordConfirm");
+const passwordConfirmError = document.querySelector(".passwordConfirmError");
+
+const form = document.querySelector("#form");
+
+function emailOnFocusOut(e) {
```

Comment:
보통 이런 핸들러함수는 On+명사+동사 또는 On+동사+명사 중에 컨벤션을 정한대로 네이밍합니다!
```suggestion
function OnFocusOutEmail(e) {
```